### DISCUSSION TITLE
Deaktivieren der Update-Funktion für einen bestimmten Zeitraum

### DISCUSSION THREAD

**Discussion by ghost** [OpenCCU/OpenCCU#2471] [🇩🇪 Ideen]
Wäre es möglich, eine neue Funktion zu implementieren, die es dem Benutzer ermöglicht, alle Updates für einen von ihm festgelegten Zeitraum zu deaktivieren? Dies schließt sowohl größere System-Updates als auch Geräte-Firmware-Updates ein.

**Comment by jens-maus** [comment]
> Wäre es möglich...

Möglich ist alles ;)

> eine neue Funktion zu implementieren, die es dem Benutzer ermöglicht, alle Updates für einen von ihm festgelegten Zeitraum zu deaktivieren? Dies schließt sowohl größere System-Updates als auch Geräte-Firmware-Updates ein.

Was soll hier genau der Hintergrund sein? Nur diese zu deaktivieren damit du davon vermeintlich nicht "genervt" wirst wenn du dich auf der WebUI einloggst? Dann kann man das doch getrost einfach ignorieren oder nicht?

**Reply by ghost** [reply]
Natürlich kann ich es ignorieren, ich habe kein Problem damit. Ich dachte nur, es wäre das i-Tüpfelchen für Verrückte wie mich, die fast jeden Tag auf dem Hauptsystem aktualisieren - mit jedem neuen Snapshot ;-) das wäre etwas Neues bei RasberryMatic, und vielleicht wird es mehr Leute interessieren, ich weiß es nicht, es ist nur meine kleine Idee 
![Bez tytułu](https://github.com/jens-maus/RaspberryMatic/assets/81189856/52f7781d-0b0c-41c1-a2cc-66b728ee1c2a)

**Reply by Baxxy13** [reply]
Ich wäre eher dafür den "Update-Branch" anpassen zu können.
Also Nightly's oder offizielles Release.

Dann könnte man auch auf den aktuellen Nightly über die WebUI updaten.

Andererseits nutze ich eh nur noch `/bin/checkFirmwareUpdate.sh`, also alles halb so wild. 😉

**Reply by ghost** [reply]
Perfect Baxxy13 😉😉😉 Danke 🍺

**Reply by ghost** [reply]
Schade, ich kann die Datei nicht ändern (checkFirmwareUpdate.sh) :-) kein Zugriff.

**Reply by jens-maus** [reply]
Da "verändert" man ja auch nix. Die Rufst du auf der Zentrale auf. Einfach SSH Zugang zur CCU/RaspberryMatic aufmachen und `checkFirmwareUpdate.sh` auf der Kommandozeile aufrufen und staunen :) Und dann natürlich `checkFirmwareUpdate.sh -h` aufrufen um die Hilfeseite zu bekommen und dann entsprechend das nutzen.

**Comment by Baxxy13** [comment]
> Schade, ich kann die Datei nicht ändern (checkFirmwareUpdate.sh)

Das hast du missverstanden. 
Das Script hat nichts mit irgendwelchen Blocks oder Unterdrückungen zu tun sondern dient dazu die RaspberryMatic per ssh zu updaten/downgraden. Also ohne über die WebUI zu gehen.

**Comment by ghost** [comment]
Alles klar, danke für die Klarstellung.
![1](https://github.com/jens-maus/RaspberryMatic/assets/81189856/ab5df511-dcba-48cc-a36f-b1629e10832d)
